services/vision/builtin/builtin.go

```diff
@@ -7,14 +7,14 @@ package builtin
 import (
```

Comment:
The vision service is being rewritten based on model service in a branch. I let the team know they'll need to rebase when this goes in.

Reply:
okay